README.md

```diff
@@ -5,7 +5,7 @@
 ![](https://img.shields.io/github/issues/hedyorg/hedy)
 
 ![](static/images/Hedy-logo-96x96-round.png)
-
+<div id="top"></div>
```

Comment:
This too is a recent change on main, and as fas as I understand should not be here.